src/components/SchemaTable/TableLayout.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { Fragment } from 'react';
+import { array } from 'prop-types';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(theme => ({
+  wrapper: {
+    display: 'block',
+    overflow: 'hidden',
+    position: 'relative',
+    backgroundColor: theme.palette.background.paper,
+  },
+
+  left: {
+    float: 'left',
+    width: '50%',
+    boxSizing: 'border-box',
+    borderTop: `${theme.spacing(0.25)}px solid ${theme.palette.divider}`,
+    overflowX: 'wrap',
+  },
+  right: {
+    float: 'left',
+    width: '50%',
+    boxSizing: 'border-box',
+    borderTop: `${theme.spacing(0.25)}px solid ${theme.palette.divider}`,
+    overflowX: 'wrap',
```

Comment:
hm, I don't think `wrap` is a valid css value. Maybe we should change this to `auto`? That would probably fix the issue with long regexes not being scrollable.

<img width="257" alt="Screen Shot 2020-09-17 at 4 18 01 PM" src="https://user-images.githubusercontent.com/3766511/93524037-ecb83480-f901-11ea-8483-c2a11197f9bf.png">

After changing it to `auto`:

<img width="251" alt="Screen Shot 2020-09-17 at 4 20 12 PM" src="https://user-images.githubusercontent.com/3766511/93524034-e9bd4400-f901-11ea-9ae7-41aa01134d97.png">

This comment applies to the class right above it as well.